Barotrauma/BarotraumaClient/ClientSource/Items/Components/ItemLabel.cs

```diff
@@ -60,6 +68,16 @@ public string Text
             }
         }
 
+        /*
+        private GUIFont font;
+        [Editable, Serialize("UnscaledSmallFont", IsPropertySaveable.Yes, description: "The label's font.")]
+        public GUIFont Font
+        {
+            get => font;
+            set => font = textBlock.Font = value;
+        }
+        */
```

Comment:
This and the next were commented out since I couldn't find a way to implement font selection.
I meant to come back to it, but I suppose I had forgot (until now).